### ISSUE TITLE
Toggle Image and URL field in creating post

### ISSUE THREAD

**Issue by trymeouteh** [liftoff-app/liftoff#325]
When you create a new post there is the URL field, image field, body field, etc.

When you select an image to upload, it will override the URL to the URL of the image hosted on the lemmy instance. This is confusing since users may thing you can have a URL and a image in a post but in reality you can only have one or the other.

I would like to suggest a toggle button were users can choose to have a image or a URL and not have both fields visible at once.

**Comment by mykdavies** [comment]
Good point. We can change the default text for that field to say "URL or Uploaded Photo" or similar, that should help.
